spring-components/rdf4j-spring/src/main/java/org/eclipse/rdf4j/spring/dao/exception/Rdf4JDaoException.java

```diff
@@ -0,0 +1,26 @@
+package org.eclipse.rdf4j.spring.dao.exception;
+
+public class Rdf4JDaoException extends Rdf4JSpringException {
```

Comment:
Minor editorial: in the existing code we tend to spell RDF4J all capitals when used in class names.

Reply:
done